### PR TITLE
Modified existed  and added three files

### PR DESCRIPTION
**Pull request by Sajj96**

### CODE REVIEW

File: Basic_syntax.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import sys
```
Comment (younggeeks) [inline Basic_syntax.py:1]: I'm not sure why you imported this sys module

**Review by younggeeks — APPROVED** [review]
LGTM

**Comment by Sajj96** [comment]
At line number seven, I used sys to print out value in x that is why I
imported sys module.

**Review by wikedzit — APPROVED** [review]
NW

**Comment by Sajj96** [comment]
Thanks brother

**Pull request merged**